code/modules/paperwork/paper.dm

```diff
@@ -468,6 +444,32 @@
 
 	add_fingerprint(user)
 	return
+	
+/obj/item/weapon/paper/proc/stamp(var/obj/item/weapon/stamp/S)
+	stamps += (!stamps || stamps == "" ? "<HR>" : "") + "<img src=large_[S.icon_state].png>"
+
+	var/image/stampoverlay = image('icons/obj/bureaucracy.dmi')
+	var/{x; y;}
```

Comment:
This is the most terrifying variable I have ever seen. It's relative-pathed variables on one line.